execution/executor-benchmark/src/transaction_generator.rs

```diff
@@ -576,6 +626,12 @@ impl TransactionGenerator {
                 connected_tx_grps,
                 shuffle_connected_txns,
             );
+        } else if maybe_hotspot_probability.is_some() {
```

Comment:
In the code somewhere assert both are not provided at the same time. (maybe as an argument group https://github.com/aptos-labs/aptos-core/blob/3e78e8a78b788f85bb84906a568e0d24242dda83/execution/executor-benchmark/src/main.rs#L168)

Reply:
fixed with `conflicts_with_all` macro, should also be fine?